# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498133390 14400
#      Thu Jun 22 08:09:50 2017 -0400
# Node ID 81b141d265a3f277aa9bc270dffb5bc61cccf75c
# Parent  1d1ce4df5255492061c63d573153861819ea1e2c
move octave-preserve-stream-state.h from libinterp to liboctave

* liboctave/util/octave-preserve-stream-state.h:
Rename from libinterp/corefcn/octave-preserve-stream-state.h.

* libinterp/corefcn/module.mk: Update.
* liboctave/util/module.mk: Update.

diff --git a/libinterp/corefcn/module.mk b/libinterp/corefcn/module.mk
--- a/libinterp/corefcn/module.mk
+++ b/libinterp/corefcn/module.mk
@@ -66,17 +66,16 @@ COREFCN_INC = \
   %reldir%/oct-prcstrm.h \
   %reldir%/oct-procbuf.h \
   %reldir%/oct-stdstrm.h \
   %reldir%/oct-stream.h \
   %reldir%/oct-strstrm.h \
   %reldir%/oct.h \
   %reldir%/octave-default-image.h \
   %reldir%/octave-link.h \
-  %reldir%/octave-preserve-stream-state.h \
   %reldir%/pager.h \
   %reldir%/pr-output.h \
   %reldir%/procstream.h \
   %reldir%/profiler.h \
   %reldir%/sighandlers.h \
   %reldir%/sparse-xdiv.h \
   %reldir%/sparse-xpow.h \
   %reldir%/symtab.h \
diff --git a/liboctave/util/module.mk b/liboctave/util/module.mk
--- a/liboctave/util/module.mk
+++ b/liboctave/util/module.mk
@@ -13,16 +13,17 @@ UTIL_INC = \
   %reldir%/lo-cutils.h \
   %reldir%/lo-hash.h \
   %reldir%/lo-ieee.h \
   %reldir%/lo-regexp.h \
   %reldir%/lo-traits.h \
   %reldir%/lo-utils.h \
   %reldir%/f77-fcn.h \
   %reldir%/lo-error.h \
+  %reldir%/octave-preserve-stream-state.h \
   %reldir%/quit.h \
   %reldir%/oct-base64.h \
   %reldir%/oct-binmap.h \
   %reldir%/oct-cmplx.h \
   %reldir%/oct-glob.h \
   %reldir%/oct-inttypes-fwd.h \
   %reldir%/oct-inttypes.h \
   %reldir%/oct-locbuf.h \
diff --git a/libinterp/corefcn/octave-preserve-stream-state.h b/liboctave/util/octave-preserve-stream-state.h
rename from libinterp/corefcn/octave-preserve-stream-state.h
rename to liboctave/util/octave-preserve-stream-state.h
